src/components/PreviewComment/index.js

```diff
@@ -145,7 +145,7 @@ export const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(12),
     lineHeight: 19.36,
-    color: COLORS.mineShaft
+    color: COLORS.greyseries
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color `COLORS.greyseries` has been replaced with `COLORS.mineShaft`. If this is intentional and `COLORS.greyseries` is equivalent to the old `COLORS.mineShaft`, then this change is fine. However, if they are not equivalent, this could potentially cause a visual discrepancy.
```diff
- color: COLORS.greyseries
+ color: colors.mineShaft
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->